pkg/events/parse_args_helpers.go

```diff
@@ -16,7 +18,7 @@ func parseSocketDomainArgument(arg *trace.Argument, domain uint64) {
 	arg.Type = "string"
 	socketDomainArgument, err := parsers.ParseSocketDomainArgument(domain)
 	if err != nil {
-		arg.Value = ""
+		arg.Value = fmt.Sprint(domain)
```

Comment:
It seems that all types are previously known, right? In this case, even being a corner case, I would recommend using `strconv.Format{Uint,Int}()` since it's a straightforward conversion which doesn't use the expensive reflection.

Take a look:

```
BenchmarkAll/Sprint_1-32         	10000000	        47.03 ns/op	      16 B/op	       2 allocs/op
BenchmarkAll/Strconv_1-32        	10000000	        16.00 ns/op	       5 B/op	       1 allocs/op
BenchmarkAll/Sprint_2-32         	10000000	        49.21 ns/op	      16 B/op	       2 allocs/op
BenchmarkAll/Strconv_2-32        	10000000	        15.59 ns/op	       5 B/op	       1 allocs/op
```